index-commercial.html

```diff
@@ -220,6 +220,13 @@ <h2>OpenShift Container Platform</h2>
               </div>
             </div>
           </div>
+          <div class="col-sm-3 text-center">
+            <h2>OpenShift Container Engine</h2>
+            <div class="list-group">
+              <p class="list-group-item-text">The OpenShift Container Engine (OCE) is the core of the OpenShift Container Platform (OCP). Use the links under the OCP docs to get relevant docs for OCE.</p>
```

Comment:
So, are OCE and OCP actually approved shortnames? I'd be happier removing both OCE and OCP from both these PRs.